drivers/timer/cortex_m_systick.c

```diff
@@ -142,7 +149,8 @@ void sys_clock_isr(void *arg)
 		 * We can assess if this is the case by inspecting COUNTFLAG.
 		 */
 
-		dticks = (cycle_count - announced_cycles) / CYC_PER_TICK;
+		dcycles = cycle_count - announced_cycles;
+		dticks = dcycles / CYC_PER_TICK;
```

Comment:
Note that this is now a 64 bit divide, which is a nontrivial change in an ISR.

Reply:
This is 32 bit divide, just as it was before